src/modules/shared/utils/i18n.ts

```diff
@@ -1,7 +1,7 @@
 import { type Locale as DateFnsLocale } from 'date-fns';
 import nlBE from 'date-fns/locale/nl-BE/index.js';
 
-export function getLocaleFromi18nLanguage(language: string): DateFnsLocale {
+export function getLocaleFromI18nLanguage(language: string): DateFnsLocale {
 	switch (language) {
 		case 'nl':
```

Comment:
Locale.NL ? , I know you did not change this part of the code, but it is convenient now you are here.